src/main/java/seedu/duke/storage/Storage.java

```diff
@@ -0,0 +1,109 @@
+package seedu.duke.storage;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.io.FileOutputStream;
+import java.io.File;
+import java.util.List;
+import seedu.duke.constants.StorageConstants;
+import seedu.duke.entries.Category;
+import seedu.duke.entries.Entry;
+import seedu.duke.util.CategoryUtil;
+
+public class Storage {
+
+    private String filePath = StorageConstants.RELATIVE_FILE_NAME;
+    private String delimiter = StorageConstants.DELIMITER;
```

Comment:
I suppose these should be `final` since they are not modified? Will be helpful in preventing any errors from surfacing during development